lib/LaTeXML/Core/Mouth.pm

```diff
@@ -214,7 +214,7 @@ sub handle_escape {    # Read control sequence
   # Bit I believe that he does NOT mean within control sequences
   my $cs = "\\" . $ch;    # I need this standardized to be able to lookup tokens (A better way???)
   if ((defined $cc) && ($cc == CC_LETTER)) {    # For letter, read more letters for csname.
-    while ((($ch, $cc) = getNextChar($self)) && $ch && ($cc == CC_LETTER)) {
+    while ((($ch, $cc) = getNextChar($self)) && (length($ch) > 0) && ($cc == CC_LETTER)) {
```

Comment:
Is this really supposed to be positive length, rather than `defined`?

Reply:
True, the length check is extra-paranoid (I believe I was triple-checking I am not misunderstanding the return values).

My fear was that the empty string `""` is defined, but empty. But studying `getNextChar`, there is no code path that could return `""`. So I can switch this to a "defined" test, thanks.

Reply:
Done. Also rebased to master - the English babel load continues to succeed.